src/main/java/net/dries007/tfc/util/registry/RegistryWood.java

```diff
@@ -31,6 +31,8 @@ public interface RegistryWood extends StringRepresentable
 
     int daysToGrow();
 
+    int autumnIndex();
```

Comment:
needs a javadoc

Reply:
Can actually be removed entirely, because there's nothing about `RegistryWood` that requires it to be exposed. It's only applied for our own blocks so it's fine as a method just on `Wood`.